.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "js-miniapp"]
-	path = js-miniapp
+	path = Sources/Classes/js-miniapp
```

Comment:
I moved `js-miniapp` submodule because it was not embedded into the SPM bundle

Reply:
just for notice 👆  the js files were not added for me in the new folder so I had to copy them manually

Reply:
> just for notice 👆 the js files were not added for me in the new folder so I had to copy them manually
Maybe a `git submodule update` is necessary